extensions/2.0/Khronos/KHR_animation_pointer/schema/glTF.KHR_animation_pointer.schema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "KHR_animation_pointer glTF extension",
```

Comment:
Since this extension is only allowed to be placed on the `animation.channel.target.schema.json` data structure, the title here should be `"KHR_animation_pointer glTF Animation Channel Target Extension"`, and the file should be named  `animation.channel.target.KHR_animation_pointer.schema.json`.

See these PRs for discussion: https://github.com/KhronosGroup/glTF/pull/2313 and https://github.com/KhronosGroup/glTF/pull/2314